### PR TITLE
improve compilation documentation

### PR DESCRIPTION
**Pull request by Thomas-Ulrich**
fixes Asagi compilation on supermuc described twice.
Removed reference to precompiles ASAGI and libxmm on supermuc (in the end, I don't think it is a significant time gain).

### CODE REVIEW

File: Documentation/asagi.rst [hunk 1]
```diff
@@ -42,49 +42,29 @@ compilation of SeisSol in a later step!
...
-   cmake FC=mpif90 CXX=mpiCC CC=mpicc .. -DCMAKE_INSTALL_PREFIX=$(pwd)/build
-   make -j8
-   make install
-
--  set the following paths
-
-.. code-block:: bash
+  mkdir build && cd build
+  CMAKE_PREFIX_PATH=$NETCDF_BASE
+  cmake .. -DSHARED_LIB=no -DSTATIC_LIB=yes -DNONUMA=on -DCMAKE_INSTALL_PREFIX=$HOME
```
Comment (sebwolf-de) [inline Documentation/asagi.rst:63]: Why `NONUMA=on`? There is a numa module on supermuc: `numactl/2.0.14-intel21`

File: Documentation/asagi.rst [hunk 2]
```diff
@@ -42,49 +42,29 @@ compilation of SeisSol in a later step!
...
 
-   module load intel intel-mpi
-   module load netcdf-hdf5-all/4.7_hdf5-1.10-intel19-impi
-   module load gcc/9
-   module load cmake/3.16.5
-
--  get the repository
-
-On a cluster without restricted access to outside sources, you could then clone the repository using:
+  git clone git@github.com:TUM-I5/ASAGI
```
Comment (sebwolf-de) [inline Documentation/asagi.rst:49]: Could use `--recursive`, saves the `update --init` later.

File: Documentation/asagi.rst [hunk 3]
```diff
@@ -42,49 +42,29 @@ compilation of SeisSol in a later step!
 example on SuperMuc
```
Comment (sebwolf-de) [inline Documentation/asagi.rst:42]: If we do not talk about modules, it is general, not SuperMUC specific, right?

**Review by sebwolf-de — CHANGES_REQUESTED** [review]
Thanks for taking care of that!
I think, after that PR, there is less redundancy. We should try to keep `compilation` as general as possible, such that anybody can install SeisSol on (almost) any system, following that guide.
The `supermuc` section should then be about pecularities of Supermuc, e.g. which modules/compilers to use.

File: Documentation/asagi.rst (hunk 2 above)
Comment (Thomas-Ulrich) [reply Documentation/asagi.rst:49]: Current approach:
```
di73yeq4@login02:~> git clone git@github.com:TUM-I5/ASAGI 
Cloning into 'ASAGI'...
Enter passphrase for key '/dss/dsshome1/0A/di73yeq4/.ssh/id_rsa': 
remote: Enumerating objects: 6953, done.
remote: Counting objects: 100% (14/14), done.
remote: Compressing objects: 100% (12/12), done.
remote: Total 6953 (delta 5), reused 9 (delta 2), pack-reused 6939
Receiving objects: 100% (6953/6953), 6.61 MiB | 1.18 MiB/s, done.
Resolving deltas: 100% (4983/4983), done.
Updating files: 100% (220/220), done.
di73yeq4@login02:~> cd ASAGI/
di73yeq4@login02:~/ASAGI> git submodule update --init
Submodule 'submodules/utils' (git@github.com:TUM-I5/utils.git) registered for path 'submodules/utils'
Cloning into '/dss/dsshome1/0A/di73yeq4/ASAGI/submodules/utils'...
Enter passphrase for key '/dss/dsshome1/0A/di73yeq4/.ssh/id_rsa': 
Submodule path 'submodules/utils': checked out 'cef6fa196e24713762a58b321f40b4c7fd3d9549'
```
with recursive (does not work with ssh):
```
di73yeq4@login02:~> git clone git@github.com:TUM-I5/ASAGI --recursive
Cloning into 'ASAGI'...
Enter passphrase for key '/dss/dsshome1/0A/di73yeq4/.ssh/id_rsa': 
remote: Enumerating objects: 6953, done.
remote: Counting objects: 100% (14/14), done.
remote: Compressing objects: 100% (12/12), done.
remote: Total 6953 (delta 5), reused 9 (delta 2), pack-reused 6939
Receiving objects: 100% (6953/6953), 6.61 MiB | 1.09 MiB/s, done.
Resolving deltas: 100% (4983/4983), done.
Updating files: 100% (220/220), done.
Submodule 'submodules/utils' (git@github.com:TUM-I5/utils.git) registered for path 'submodules/utils'
Cloning into '/dss/dsshome1/0A/di73yeq4/ASAGI/submodules/utils'...
Enter passphrase for key '/dss/dsshome1/0A/di73yeq4/.ssh/id_rsa': 
remote: Enumerating objects: 2399, done.        
remote: Counting objects: 100% (17/17), done.        
remote: Compressing objects: 100% (12/12), done.        
remote: Total 2399 (delta 7), reused 12 (delta 5), pack-reused 2382        
Receiving objects: 100% (2399/2399), 1.49 MiB | 1.02 MiB/s, done.
Resolving deltas: 100% (2132/2132), done.
Submodule path 'submodules/utils': checked out 'cef6fa196e24713762a58b321f40b4c7fd3d9549'
Submodule 'submodules/cxxtest' (https://github.com/CxxTest/cxxtest.git) registered for path 'submodules/utils/submodules/cxxtest'
Cloning into '/dss/dsshome1/0A/di73yeq4/ASAGI/submodules/utils/submodules/cxxtest'...
fatal: unable to access 'https://github.com/CxxTest/cxxtest.git/': Failed to connect to github.com port 443: Network is unreachable
fatal: clone of 'https://github.com/CxxTest/cxxtest.git' into submodule path '/dss/dsshome1/0A/di73yeq4/ASAGI/submodules/utils/submodules/cxxtest' failed
Failed to clone 'submodules/cxxtest'. Retry scheduled
Cloning into '/dss/dsshome1/0A/di73yeq4/ASAGI/submodules/utils/submodules/cxxtest'...
fatal: unable to access 'https://github.com/CxxTest/cxxtest.git/': Failed to connect to github.com port 443: Network is unreachable
fatal: clone of 'https://github.com/CxxTest/cxxtest.git' into submodule path '/dss/dsshome1/0A/di73yeq4/ASAGI/submodules/utils/submodules/cxxtest' failed
Failed to clone 'submodules/cxxtest' a second time, aborting
Failed to recurse into submodule path 'submodules/utils'
```

Comment (sebwolf-de) [reply Documentation/asagi.rst:45]: Okay, then we keep it as is...

**Review by sebwolf-de — APPROVED** [review]

**Pull request merged**